pages/_en-US/community/discord-rules.md

```diff
@@ -26,18 +26,19 @@ Examples of what breaks this rule include:
    - Important information can be found in [#useful-resources](https://discord.com/channels/283769550611152897/638041441079263283) and in the pinned messages of all the channels
 - Taking up excessive space in chat (e.g. Sending a tiny amount of words in multiple messages instead of keeping it all in one message)
 - Repeating messages, whether it is repeated in the same channel or in multiple channels
- - If you're unsure where to post your message check the channel topics and ask in the most fitting one
-   - If you're still unsure, ask in [#community-meta](https://discord.com/channels/283769550611152897/715651368391671919)
+ - If you're unsure where to post your message, identify which channel would be the most fitting for your message by check its topic. Alternatively, ask in [#community-meta](https://discord.com/channels/283769550611152897/715651368391671919)
 - Posting for the sake of posting rather than to bring actual conversations (such as complaining that the chat is dead, only sending out greetings, copypastas, etc)
-- Sending extremely long messages, use a tool such as [mystbin](https://mystb.in/) for longer messages instead
+- Sending extremely long messages, which ruins the multiple-user aspect of having a conversation. Use a tool such as [mystbin](https://mystb.in/) for longer messages instead, which would put your messages externally, allowing people to read others' messages.
```

Comment:
As mentioned on Discord, would it be possible to have a Discord bot automatically filter messages surpassing a certain length, copy and paste them in something like mystbin or "Samplasion's one" as suggested by NightScript, and replacing that rant with a link to that. Then after that provide the user with a warning about ranting.

One issue with this would be the rant being divided in several short messages in quick bursts.

Reply:
Well, the rules about long messages and rant based messages are unrelated. We don't want to deal with either of them, even if the rants are short or the other convo is long.

When I suggested it, I was moreso thinking of something that would have 20 newline characters or so.